gradle-plugin/src/main/kotlin/io/spine/protodata/gradle/plugin/LaunchProtoData.kt

```diff
@@ -190,3 +193,31 @@ private fun Provider<List<Directory>>.absoluteDirs() = takeIf { it.isPresent }
 
 private fun Provider<List<Directory>>.absolutePaths(): String =
     absoluteDirs().joinToString(pathSeparator)
+
+/**
+ * Tells if the request file for this task exists.
+ *
+ * Logs error if the given source set contains `proto` directory which contains files,
+ * which assumes that the request file should have been created.
+ */
+internal fun LaunchProtoData.checkRequestFile(sourceSet: SourceSet): Boolean {
+    val requestFile = requestFile.get().asFile
+    if (!requestFile.exists() && sourceSet.containsProtoFiles()) {
+        project.logger.error(
+            "The task `${name}` was skipped because" +
```

Comment:
"Task ... was skipped" sounds not as an error description, but rather like a warning. 

I suggest starting with something like "Error locating the request file...".

Reply:
@alexander-yevsyukov I think you might have missed this suggestion.

Reply:
Yes, you're right. Was reviewing from the mobile app, and it wasn't visible. I'm going to address it in a next PR.